src/main/scala/com/campudus/tableaux/Starter.scala

```diff
@@ -1,32 +1,49 @@
 package com.campudus.tableaux
 
+import com.campudus.tableaux.router.TableauxRouter
 import org.vertx.scala.core.FunctionConverters._
 import org.vertx.scala.core.http.HttpServer
-import org.vertx.scala.core.json.Json
-import org.vertx.scala.platform.Verticle
+import org.vertx.scala.core.json.{Json, JsonObject}
+import org.vertx.scala.platform.{Container, Verticle}
 
 import scala.concurrent.{Future, Promise}
-import scala.util.{Try, Failure, Success}
+import scala.util.{Failure, Success, Try}
 
 class Starter extends Verticle {
 
   val DEFAULT_PORT = 8181
-  val config = Json.obj("username" -> "postgres", "password" -> "admin")
+  val DEFAULT_DATABASE_ADDRESS = "campudus.asyncdb"
```

Comment:
Actually, we could put these constants into an `object`. We could use this value in our tests then, too.